app/pathfinder_svc.py

```diff
@@ -31,8 +30,8 @@ async def import_scan(self, scan_format, report):
             temp_file = '%s/_temp_report_file.tmp' % settings.data_dir
             with open(temp_file, 'wb') as f:
                 f.write(contents)
-            parsed_report = self.parsers[scan_format].parse(temp_file, report)
-            #parsed_report = self.enrich_report(parsed_report)
+            parsed_report = self.parsers[scan_format].parse(temp_file)
+            parsed_report = self.enrich_report(parsed_report)
```

Comment:
Remind me, why wasnt this being called previously and now is?

Reply:
That was an accident during upload. I had it uncommented locally.